src/devices/solar.cpp

```diff
@@ -406,4 +486,15 @@ void Solar::process_ISM1Set(std::shared_ptr<const Telegram> telegram) {
     changed_ |= telegram->read_value(setpoint_maxBottomTemp_, 6);
 }
 
+bool Solar::set_SM100Tank1MaxTemp(const char * value, const int8_t id) {
+    int temp;
```

Comment:
the types are important as they're used in the Template to render the version. Please use int8_t, uin8_t, init16_t etc

Reply:
Interestingly, when I replace int temp; with int8_t temp; I get a compile error. Helpers::value2number takes an actual int.